src/components/ui/Lecture/Lecture.module.scss

```diff
@@ -0,0 +1,63 @@
+.lecturer {
+  &__heading {
+    color: var(--color-text-strong);
+    margin-bottom: var(--space-m);
+  }
+
+  &__bio {
+    display: flex;
+    gap: var(--space-l);
+
+    @media (max-width: 760px) {
+      flex-direction: column;
+    }
+  }
+
+  &__image {
+    & > div {
+      display: block;
+      width: 224px;
+      min-width: 224px;
+      height: 224px;
+      min-height: 224px;
+    }
+
+    & img {
+      width: 100%;
+      height: 100%;
+      object-fit: cover;
+      border-radius: 4px 16px;
+    }
+
+    &--scale {
+      object-fit: scale-down !important;
```

Comment:
ojojoj co to za important

Reply:
Ciii, żeby nikt więcej nie zobaczył 🤫 Poprawimy po wydarzeniu XD Najważniejsze, że działa xd